docs/source/60_building_apps.md

```diff
@@ -256,6 +256,31 @@ $ spark-submit \
 A complete example of this pattern can be found in the
 [heuermh/adam-examples](https://github.com/heuermh/adam-examples) repository.
 
+### Using ADAM’s RegionJoin API
+Another useful API implemented in ADAM is the RegionJoin API, which joins two genomic datasets that contain overlapping regions. This primitive is useful for a number of applications including variant calls (identifying all of the reads that overlap a candidate variant), coverage analysis (determining the coverage depth for each region in a reference), and indel realignment (identify indels aligned against a reference).
+
+There are two implementations of joins available in ADAM: BroadcastRegionJoin and ShuffleRegionJoin. The result of a ShuffleRegionJoin is identical to the BroadcastRegionJoin, however they should each be used in specific circumstances. The ShuffleRegionJoin performs a copartition on the right dataset before the join on the entire dataset, but the BroadcastRegionJoin sends a copy of the entire right dataset to each node. ShuffleRegionJoin should be used in the case that the right dataset is too large to send to all nodes or if there is high coverage skew in either dataset. The BroadcastRegionJoin should be used when you are joining a smaller dataset to a larger one.
```

Comment:
* "implementations of joins" -> "join implementations"
* "ShuffleRegionJoin should be used in the case" -> "ShuffleRegionJoin should be used if"
* RE: "if there is high coverage skew in either dataset," this is not necessarily true. Broadcast joins work really well if your left side has hot keys _and_ a randomized partitioning.